packages/plugins/sentry/server/services/__tests__/sentry.test.js

```diff
@@ -87,7 +87,7 @@ describe('Sentry service', () => {
     expect(mockCaptureException).toHaveBeenCalled();
   });
 
-  it('does not not send metadata when the option is disabled', () => {
+  it('does not send metadata when the option is disabled', () => {
```

Comment:
This isn't precisely a spelling error, but old fashioned spell checking tools flag such things because it's easy, so I've recently included it.